pkg/scheduler/plugins/conformance/conformance_test.go

```diff
@@ -0,0 +1,66 @@
+package conformance
+
+import (
+	"reflect"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+
+	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/conf"
+	"volcano.sh/volcano/pkg/scheduler/framework"
+	"volcano.sh/volcano/pkg/scheduler/uthelper"
+	"volcano.sh/volcano/pkg/scheduler/util"
+)
+
+func TestConformancePlugin(t *testing.T) {
+	var (
+		// prepare preemptor and preemptees
+		task1 = api.NewTaskInfo(
+			util.BuildPod("kube-system", "test-pod",
+				"test-node", v1.PodRunning, api.BuildResourceList("1", "1Gi"), "pg1", make(map[string]string), make(map[string]string)),
+		)
+		task2 = api.NewTaskInfo(
+			util.BuildPod("test-namespace", "test-pod",
+				"test-node", v1.PodRunning, api.BuildResourceList("1", "1Gi"), "pg1", make(map[string]string), make(map[string]string)),
+		)
+	)
+	tests := []struct {
+		uthelper.TestCommonStruct
+		arguments  framework.Arguments
+		preemptees []*api.TaskInfo
+	}{
+		{
+			TestCommonStruct: uthelper.TestCommonStruct{
+				Name:    "conformance plugin",
+				Plugins: map[string]framework.PluginBuilder{PluginName: New},
+			},
+		},
+	}
+
+	for _, test := range tests {
+		trueValue := true
+		tiers := []conf.Tier{
+			{
+				Plugins: []conf.PluginOption{
+					{
+						Name:               PluginName,
+						EnabledPreemptable: &trueValue,
+						Arguments:          test.arguments,
+					},
+				},
+			},
+		}
+		t.Run(test.Name, func(t *testing.T) {
+			ssn := test.RegisterSession(tiers, nil)
+			preemptees := []*api.TaskInfo{task1, task2}
```

Comment:
We should put preeemptees and expected preemptees in tests struct so that it is easy to extend other cases..

Reply:
thanks! done